Blender/src/babylon_js/mesh.py

```diff
@@ -167,7 +167,11 @@ def __init__(self, bpyMesh, scene, exporter):
                 Logger.warn('No materials have been assigned: ', 2)
 
         # Get mesh temporary version of mesh with modifiers applied
-        mesh = bpyMesh.to_mesh(bpy.context.depsgraph, True)
+        if bpyMesh.type == 'MESH':
+            # mesh = bpyMesh.to_mesh(bpy.context.depsgraph, True) # old

```

Comment:
Maybe it is not necessary to have this `# old `line here?

Reply:
No, you don't need that commented line, you can erase it.

Reply:
Would you update your branch with a commit that deletes this line?